### PR TITLE
Improve the grammar and annotations 

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Improve the quality of annotations.

### CODE REVIEW

**Comment by afbjorklund** [comment]
While I approve spelling Kubernetes “properly” in the user-facing documentation, I find it disturbing to not be able to freely say “kubernetes cluster” in developer documentation and comments. It breaks the reading flow, even if it is inconsistent to have two spellings.

Just my 2 cents.

**Comment by yuxiaobo96** [comment]
> While I approve spelling Kubernetes “properly” in the user-facing documentation, I find it disturbing to not be able to freely say “kubernetes cluster” in developer documentation and comments. It breaks the reading flow, even if it is inconsistent to have two spellings.
> 
> Just my 2 cents.

**Comment by yuxiaobo96** [comment]
Sorry, I don't understand what you mean.

**Comment by yuxiaobo96** [comment]
@hwdef

**Comment by hwdef** [comment]
/ok-to-test
LGTM

**Comment by medyagh** [comment]
thank you so much for fixing and improving the docs !

> Sorry, I don't understand what you mean.

@afbjorklund meant the capitalization is good but not everywhere. I will comment on the changes for more clarity.

File: pkg/minikube/machine/cache_binaries.go [hunk 1]
```diff
@@ -51,12 +51,12 @@ func CacheBinariesForBootstrapper(version string, clusterBootstrapper string) er
...
 }
 
-// KubernetesReleaseURL gets the location of a kubernetes client
+// KubernetesReleaseURL gets the location of a Kubernetes client
 func KubernetesReleaseURL(binaryName, version, osName, archName string) string {
 	return fmt.Sprintf("https://storage.googleapis.com/kubernetes-release/release/%s/bin/%s/%s/%s", version, osName, archName, binaryName)
 }
 
-// KubernetesReleaseURLSHA1 gets the location of a kubernetes client checksum
+// KubernetesReleaseURLSHA1 gets the location of a Kubernetes client checksum
```
Comment (medyagh) [inline pkg/minikube/machine/cache_binaries.go]: all the not-user facing one should be changed to kubernetes

File: pkg/minikube/config/config.go [hunk 2]
```diff
@@ -132,12 +132,12 @@ func GetMachineName() string {
 	return viper.GetString(MachineProfile)
 }
 
-// Load loads the kubernetes and machine config for the current machine
+// Load loads the Kubernetes and machine config for the current machine
```
Comment (medyagh) [inline pkg/minikube/config/config.go]: like comments, no need to capitalize kubernetes

**Comment by yuxiaobo96** [comment]
I will restore them for a while.

File: deploy/addons/helm-tiller/README.md [hunk 3]
```diff
@@ -8,16 +8,16 @@ To enable this addon, simply run:
 minikube addons enable helm-tiller
 ```
 
-In a minute or so tiller will be installed into your cluster. You could run `helm init` each time you create a new minikube instance or you could just enable this addon.
-Each time you start a new minikube instance tiller will be automatically installed. 
+In a minute or so tiller will be installed into your cluster. You could run `helm init` each time when you create a new minikube instance or you could just enable this addon.
```
Comment (bboreham) [inline deploy/addons/helm-tiller/README.md]: This one is incorrect.

Comment (yuxiaobo96) [reply deploy/addons/helm-tiller/README.md]: Please tell me in detail.
Thank you.

Comment (bboreham) [reply deploy/addons/helm-tiller/README.md]: I'm not clear what detail you want; it just sounds wrong.

Compare the [relative occurrence in books](
https://books.google.com/ngrams/graph?content=each+time+you%2Ceach+time+when+you&year_start=1900&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Ceach%20time%20you%3B%2Cc0%3B.t1%3B%2Ceach%20time%20when%20you%3B%2Cc0#t1%3B%2Ceach%20time%20you%3B%2Cc0%3B.t1%3B%2Ceach%20time%20when%20you%3B%2Cc0) of "each time you" and "each time when you".

Comment (yuxiaobo96) [reply deploy/addons/helm-tiller/README.md]: Ok.

**Comment by yuxiaobo96** [comment]
/assign @tstromberg

Thank you!

**Comment by yuxiaobo96** [comment]
/assign @sharifelgamal 

Thanks.

**Comment by medyagh** [comment]
looks good ! thank you.

**Pull request merged**